near-sdk/Cargo.toml

```diff
@@ -34,15 +34,17 @@ wee_alloc = { version = "0.4.5", default-features = false, optional = true }
 once_cell = { version = "1.17", default-features = false }
 
 near-abi = { version = "0.4.0", features = ["__chunked-entries"], optional = true }
-near-account-id = { version="1.0.0-alpha.4", features = ["serde", "borsh"] }
+near-account-id = { version="1.0.0", features = ["serde", "borsh"] }
 near-gas = { version = "0.2.3", features = ["serde", "borsh"] }
 near-token = { version = "0.2.0", features = ["serde", "borsh"] }
 
+
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-near-vm-logic = { version = "0.17", optional = true }
-near-primitives-core = { version = "0.17", optional = true }
-near-primitives = { version = "0.17", optional = true }
-near-crypto = { version = "0.17", optional = true }
+near-vm-runner = { version = "0.19", default_features = false, optional = true }
```

Comment:
@frol any way to disable `wasmer-runtime-core` build?
```bash
   Compiling near-vm-runner v0.19.0 Running `CARGO=/home/user/.rustup/toolchains/
     stable-x86_64-unknown-linux-gnu/bin/cargo CARGO_CRATE_NAME=near_vm_runner CARGO_MANIFEST_DIR=/
     home/user/.cargo/registry/src/index.crates.io-6f17d22bba15001f/near-vm-runner-0.19.0
     CARGO_PKG_AUTHORS='Near Inc <hello@nearprotocol.com>' CARGO_PKG_DESCRIPTION='This
     crate implements the specification of the interface that Near blockchain exposes to
     the smart contracts.' CARGO_PKG_HOMEPAGE='' CARGO_PKG_LICENSE='MIT OR Apache-2.0'
     CARGO_PKG_LICENSE_FILE='' CARGO_PKG_NAME=near-vm-runner CARGO_PKG_README=README.md
     CARGO_PKG_REPOSITORY='https://github.com/near/nearcore' CARGO_PKG_RUST_VERSION=''
     CARGO_PKG_VERSION=0.19.0 CARGO_PKG_VERSION_MAJOR=0 CARGO_PKG_VERSION_MINOR=19
     CARGO_PKG_VERSION_PATCH=0 CARGO_PKG_VERSION_PRE='' LD_LIBRARY_PATH='/home/user/
     Documents/code/rust_work_repos/near/near-sdk-rs/target/debug/deps:/home/user/.rustup/
     toolchains/stable-x86_64-unknown-linux-gnu/lib:/home/user/.rustup/toolchains/
     stable-x86_64-unknown-linux-gnu/lib:/nix/store/8xgb8phqmfn9h971q7dg369h647i1aa0-zlib-1.3/
     lib' /home/user/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin/rustc
     --crate-name near_vm_runner --edition=2021 /home/user/.cargo/registry/src/
     index.crates.io-6f17d22bba15001f/near-vm-runner-0.19.0/src/lib.rs --error-format=json
     --json=diagnostic-rendered-ansi,artifacts,future-incompat --crate-type lib
     --emit=dep-info,metadata,link -C embed-bitcode=no -C debuginfo=2 '--allow=clippy::all'
     --deny=warnings '--deny=clippy::suspicious' '--deny=clippy::redundant_clone'
     '--deny=clippy::len_zero' '--deny=clippy::derivable_impls' '--deny=clippy::correctness'
     '--deny=clippy::clone_on_copy' --cfg 'feature="anyhow"' --cfg 'feature="default"'
     --cfg 'feature="near-vm-compiler"' --cfg 'feature="near-vm-compiler-singlepass"'
     --cfg 'feature="near-vm-engine"' --cfg 'feature="near-vm-types"' --cfg
     'feature="near-vm-vm"' --cfg 'feature="near_vm"' --cfg 'feature="wasmer-compiler"'
     --cfg 'feature="wasmer-compiler-singlepass"' --cfg 'feature="wasmer-engine"'
     --cfg 'feature="wasmer-engine-universal"' --cfg 'feature="wasmer-runtime"' --cfg
     'feature="wasmer-runtime-core"' --cfg 'feature="wasmer-types"' --cfg 'feature="wasmer-vm"'
     --cfg 'feature="wasmer0_vm"' --cfg 'feature="wasmer2_vm"' --cfg 'feature="wasmtime"' --cfg
     'feature="wasmtime_vm"' -C metadata=e481b2a995e931cd -C extra-filename=-e481b2a995e931cd
```

Reply:
checked that changes from https://github.com/near/nearcore/pull/10423 alleviate the problem with https://github.com/near/near-sdk-rs/pull/1130/commits/22aedbf10d97d9f3c6470eed6fbd5c4e6ff3b7ad